tests/store/artifact/test_s3_artifact_repo.py

```diff
@@ -33,29 +42,31 @@ def teardown_function():
         del os.environ["MLFLOW_S3_UPLOAD_EXTRA_ARGS"]
 
 
-def test_file_artifact_is_logged_and_downloaded_successfully(s3_artifact_root, tmp_path):
+def test_file_artifact_is_logged_and_downloaded_successfully(s3_artifact_repo, tmp_path):
     file_name = "test.txt"
     file_path = os.path.join(tmp_path, file_name)
     file_text = "Hello world!"
 
     with open(file_path, "w") as f:
         f.write(file_text)
 
-    repo = get_artifact_repository(posixpath.join(s3_artifact_root, "some/path"))
+    repo = s3_artifact_repo
     repo.log_artifact(file_path)
     downloaded_text = open(repo.download_artifacts(file_name)).read()
     assert downloaded_text == file_text
 
 
-def test_file_artifact_is_logged_with_content_metadata(s3_artifact_root, tmp_path):
+def test_file_artifact_is_logged_with_content_metadata(
+    s3_artifact_repo, s3_artifact_root, tmp_path
+):
     file_name = "test.txt"
     file_path = os.path.join(tmp_path, file_name)
     file_text = "Hello world!"
 
     with open(file_path, "w") as f:
         f.write(file_text)
 
-    repo = get_artifact_repository(posixpath.join(s3_artifact_root, "some/path"))
+    repo = s3_artifact_repo
```

Comment:
Can we remove this aliasing and just use `s3_artifact_repo` directly? (and all other occurrences)

Reply:
The reason we do this aliasing / fixture is that this will automatically test for both the normal `S3ArtifactRepository` and the `OptimizedS3ArtifactRepository`. Here's the code for the fixture:
```
@pytest.fixture(params=[True, False])
def s3_artifact_repo(s3_artifact_root, request):
    if request.param:
        return OptimizedS3ArtifactRepository(posixpath.join(s3_artifact_root, "some/path"))
    return S3ArtifactRepository(posixpath.join(s3_artifact_root, "some/path"))
```

Reply:
I think what @BenWilson2's asking is remove `repo` and directly use `s3_artifact_repo`.

Reply:
Alright, done